modules/aws-eks-managed-node-groups/managed-launch-templates.tf

```diff
@@ -34,9 +34,9 @@ resource "aws_launch_template" "managed_node_groups" {
   }
 
   metadata_options {
-    http_endpoint               = var.context.http_endpoint
-    http_tokens                 = var.context.http_tokens
-    http_put_response_hop_limit = var.context.http_put_response_hop_limit
+    http_endpoint               = try(var.context.http_endpoint, "enabled")
+    http_tokens                 = try(var.context.http_tokens, "required") #tfsec:ignore:aws-autoscaling-enforce-http-token-imds
```

Comment:
tfsec wants `http_tokens` to be `required` but I don't know the consequences of enforcing that on all users. Instead, adding the `try()` block to default to a sane security setting for the IMDS and an ignore rule to silence the error